src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlInternalConnectionTds.cs

```diff
@@ -367,8 +367,7 @@ internal SqlConnectionTimeoutErrorInternal TimeoutErrorInternal
                 SessionData reconnectSessionData = null,
                 bool applyTransientFaultHandling = false,
                 string accessToken = null,
-                DbConnectionPool pool = null,
-                SqlAuthenticationProviderManager sqlAuthProviderManager = null
+                DbConnectionPool pool = null
```

Comment:
I'm not sure about these and the changes in TdsParser.cs. Why was `SqlAuthenticationProviderManager` previously allowed to be passed in instead of just referencing the Instance? I don't see how this change was necessary. It almost looks like just removing unneeded code. But I feel like I'm missing something.

Reply:
This is just unused parameter, no calling method sets it and it's always passed as null.